codeflash/optimization/function_optimizer.py

```diff
@@ -1086,8 +1086,8 @@ def run_and_parse_tests(
                     pytest_cmd=self.test_cfg.pytest_cmd,
                     pytest_timeout=INDIVIDUAL_TESTCASE_TIMEOUT,
                     pytest_target_runtime_seconds=testing_time,
-                    pytest_min_loops=1,
-                    pytest_max_loops=1,
+                    pytest_min_loops=pytest_min_loops,
```

Comment:
we can skip these arguments here since these are actually not used and is hardcoded. This is giving the wrong indication to the reader of this code, about how many times this code loops